packages/core-react/README.md

```diff
@@ -0,0 +1,13 @@
+# [Ray React Components](https://ray.wework.com/)
+
+Resources for building interfaces with WeWork's design system.
```

Comment:
since this is open source, should probably add a disclaimer that this is a work in progress and probably not production ready yet

Reply:
Should be denoted by the "alpha" release tag